inc/classes/busting/class-facebook-sdk.php

```diff
@@ -100,6 +100,12 @@ public function replace_url( $html ) {
 
 		$html      = str_replace( $tag, $replace_tag, $html );
 		$file_path = $this->get_busting_file_path( $locale );
+		$xfbml     = $this->get_xfbml_from_url( $tag );
+		$app_id    = $this->get_appId_from_url( $tag );
+		$version   = $this->get_version_from_url( $tag );
+
+		// Add FB async init.
+		$html .= '<script>window.fbAsyncInit = function fbAsyncInit () {FB.init({appId: \'' . $app_id . '\',xfbml: ' . $xfbml . ',version: \'' . $version . '\'})}</script>';
```

Comment:
The script should be inserted at a specific place in the HTML markup, like just before the `</body>` tag.

Also, we're potentially inserting variables with a false value, so the script will not work correctly in this case. We should only insert it if all values are ok.

Reply:
I think only APPID needs to be validated. In rest I can set default values for xfbml & version. 
And where this should be placed?